Bluette1/calculator
# Feature/test math magicians website

##### Feature/test math magicians website
 - Setup tests for the project.
 - Add unit tests for the `calculate.js` file.
- Test all the website's routes (by using jest snapshots ) at
    - Home page
    - Calculator page
    - Quote page 

- Make sure that all functionalities of the calculator are tested by using `react testing library`.

[Live Demo](https://mcalc-app.herokuapp.com/)
